pandas/tseries/index.py

```diff
@@ -762,6 +762,8 @@ def astype(self, dtype):
             return self.asi8.copy()
         elif dtype == _NS_DTYPE and self.tz is not None:
             return self.tz_convert('UTC').tz_localize(None)
+        elif dtype == str:
+            return self._shallow_copy(values=self.format(), infer=True)
```

Comment:
@jreback I think you added it when we were working on the `PeriodIndex` issues? https://github.com/pydata/pandas/blob/master/pandas/core/index.py#L372